certora/specs/CER-40-AccountStatusCheck/CER-42-AccountStatusCheck-NoController.spec

```diff
@@ -0,0 +1,16 @@
+methods {
+    function numOfController(address account) external returns (uint8) envfree;
+    function checkAccountStatus(address account) external returns (bool) envfree;
+}
+
+// CER-42: Account Status Check no controller
+// If there is no Controller enabled for an Account at the time of the Check, 
+// the Account Status MUST always be considered valid. It includes disabling 
+// the only enabled Controller before the Checks.
+rule account_status_no_controller {
+    env e;
+    address account;
+    require numOfController(account) == 0;
+    bool isValid = checkAccountStatus(account);
+    assert isValid;
```

Comment:
maybe it should also not revert? might be hard to prove in that. but with rule_sanity we know that it is feasible